browser-switch/src/test/java/com/braintreepayments/browserswitch/test/TestFragmentActivity.java

```diff
@@ -0,0 +1,6 @@
+package com.braintreepayments.browserswitch.test;
+
+import android.support.v4.app.FragmentActivity;
+
+public class TestFragmentActivity extends FragmentActivity {
```

Comment:
Maybe helpful to add a comment here that this new class is used for tests and is intentionally left empty?

Reply:
Hmm we could, but it would only be visible for folks currently developing. 

It's in the `src/test/` directory and those files don't get published.